src/common/header/HeaderNav.jsx

```diff
@@ -94,7 +94,7 @@ const HeaderNav = ({ showBrowse }) => {
               <a
                 className="app-header-btn app-header-btn--primary umami--click--create-play"
                 data-testid="create-btn"
-                href="https://github.com/reactplay/react-play/blob/main/CREATE-PLAY.md"
+                href="https://docs.reactplay.io/How-To-Guides/how-to-create-play"
```

Comment:
@supminn, I would recommend moving this URL to const and using it everywhere, so if we are changing the URL, we have to make the change in one place only.

Reply:
> sure, that seems like a better approach.
> 
> One question - On the docs page, I changed the link for `discussions` and added our discord link instead. Should I make this change on these files as well? (I found a few references to discussions such as [https://github.com/reactplay/react-play/discussions/new?category=ideas&title=${idea.title}](https://github.com/reactplay/react-play/discussions/new?category=ideas&title=$%7Bidea.title%7D))

No, I think that is intended to start a new discussion on the repo.